core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -54,8 +54,10 @@ class ReplicaFetcherThread(name: String,
   type REQ = FetchRequest
   type PD = PartitionData
 
+  private val logContext = new LogContext(s"[ReplicaFetcherThread brokerId=${brokerConfig.brokerId}] ")
```

Comment:
This could be `s"[ReplicaFetcher brokerId=${brokerConfig.brokerId}, sourceBrokerId=${sourceBroker.id}, fetcherId=$fetcherId] ")` and we can then remove mentions of the source broker from individual log statements.